studio/components/interfaces/Settings/General/DeleteProjectPanel/DeleteProjectButton.tsx

```diff
@@ -68,31 +69,20 @@ const DeleteProjectButton = ({ type = 'danger' }: DeleteProjectButtonProps) => {
       })
     }
 
-    setLoading(true)
-    try {
-      const response = await delete_(`${API_URL}/projects/${projectRef}`)
-      if (response.error) throw response.error
-      await invalidateProjectsQuery(queryClient)
-      ui.setNotification({ category: 'success', message: `Successfully deleted ${project.name}` })
-      router.push(`/projects`)
-    } catch (error: any) {
-      setLoading(false)
-      ui.setNotification({
-        category: 'error',
-        message: `Failed to delete project ${project.name}: ${error.message}`,
-      })
-    }
-
-    // Submit exit survey to Hubspot for paid projects
+    await deleteProject({ projectRef: project.ref })
```

Comment:
Still need a try/catch?

Reply:
oh good point!

Reply:
just pushed - gonna wrap the 2 mutations in one try catch